package.json

```diff
@@ -46,8 +46,8 @@
       "^.+\\.jsx?$": "babel-jest"
     },
     "moduleNameMapper": {
-      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "../../__mocks__/fileMock.js",
-      "\\.(css|less)$": "../../__mocks__/styleMock.js"
+      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
+      "\\.(css|less)$": "<rootDir>/__mocks__/styleMock.js"
```

Comment:
Can see the exact place where your tests actually tried to use styleMock/fileMock, this happened to me too haha

Reply:
Everytime I implement significant changes I seem to circle back around to package.json or some config file to keep components happy. Slow process!